### DISCUSSION TITLE
Welcome to tamperchrome Discussions!

### DISCUSSION THREAD

**Discussion by sirdarckcat** [google/tamperchrome#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tchoua08** [comment]
Hello Sir,
I have the tamper dev source code  v2  and i run it with ng serve 
when i test it on localhost  i didn't see something load inside the tamper dev
for exemple  i wrote [www.google.com](http://www.google.com/) inside google chrome and inside tamper dev load on localhost:4200  nothing happen.

Please Sir how do it works on localhost:4200 with ng serve 

Regards,
Bienvenue TCHOUA
